### ISSUE TITLE
Scrolling issue in Firefox

### ISSUE THREAD

**Issue by fenisteel** [olifolkerd/tabulator#2146]
There are 2 bugs with scrolling in Firefox. 

1. When using VirtualDOM if the tabulator has a height of at least 400px you can't scroll to the end of it in Firefox. See the following screenshot:

![tabulator1](https://user-images.githubusercontent.com/8095121/59389505-bdb22b00-8d6e-11e9-9709-4545ae762aa9.png)

I don't know if it has to be exactly 400px or something else on your computer, please test with various heights. In any Chromium based browser it works fine. 

2. If you turn off the VirtualDom rendering, there is no scrollbar at all: 

![tabulator2](https://user-images.githubusercontent.com/8095121/59390154-ba1fa380-8d70-11e9-9514-f010384b20a5.png)

But if you add these CSS rules to the "tabulator-table" class:
```
  height: 475px; /* This is the whole height minus the header */
  width: 100%;
  overflow: auto; /* It was set to visible */

```
Then the scrollbar is back.

Test environment: i5-4690, 8 GB RAM, Radeon R9 280, Windows 7 x64 (up-to-date), all drivers are up-to-date, Firefox 67.0.2 x64. Tested with a clean Firefox profile too.

Tabulator version: 4.2.7

[Test fiddle](https://jsfiddle.net/fenistil/hyq95ap1/12/)

**Comment by tentus** [comment]
Probably related to https://github.com/olifolkerd/tabulator/issues/2037 or https://github.com/olifolkerd/tabulator/issues/2117

If you add `table.redraw()` at the end of your fiddle and remove the two virtualDom configs it'll probably work as expected.

**Comment by fenisteel** [comment]
Thanks for the reply, I've tested it but it's the same. : (

[Here is the updated fiddle](https://jsfiddle.net/fenistil/hyq95ap1/15/)

**Comment by tentus** [comment]
Ah, your issue now is that `table` != `Table`. javascript variables are case-sensitive.

**Comment by fenisteel** [comment]
Yes I know, it was just a typo. It seems that the first problem is solved, but in the second case I still don't have a scrollbar. It not so important for now, but it's still a bug if someone needed it. Thanks for your reply.

**Comment by olifolkerd** [comment]
As a general rule you should never need to change the buffer, people really need to stop doing this it is extremely bad advice.

The virtualDomBuffer, it automatically adjusting and will sort itself out in most circumstances. it is only included for some very specific edge cases.

In most cases if you are having to manually fudge this it is because you are not redrawing your table when it changes height.

@tentus im just going to tag you in this comment as ive seen you suggest it a couple of times and i would rather this was not handed out as a solution as it leads people down the wrong path 90% of the time.

Cheers

Oli :)

**Comment by tentus** [comment]
@olifolkerd I went looking for places I suggested this so I could retract it, for the sake of future users looking for help, but the only place I could find was https://github.com/olifolkerd/tabulator/issues/2117, is there another place that you're aware of that I had suggested it? My personal opinion even prior to your comment that it was the kind of thing that should only be tampered with in unusual cornercases.

I did mention it in https://github.com/olifolkerd/tabulator/issues/2146 and https://github.com/olifolkerd/tabulator/issues/2375, but in those cases I was suggesting removing it and the value was unreasonably small, respectively, so it seems unlikely they were what you are referencing.
